tests/wikidata/test_extract.py

```diff
@@ -185,10 +234,9 @@ def mocked_item_details_response():
         lambda self, _: mocked_item_details_response(),
     )
 
-    search_results = list(search_organization_by_label(item_label="TEST"))
+    search_results = search_organization_by_label(item_label="TEST")
```

Comment:
```suggestion
    search_result = search_organization_by_label(item_label="TEST")
```

Reply:
updated in 108989566acf52f31953594e8d8cbd61be22be7a